plugin-locationlayer/src/main/java/com/mapbox/mapboxsdk/plugins/locationlayer/LocationLayerPlugin.java

```diff
@@ -526,19 +481,6 @@ private void disableLocationLayerPlugin() {
     locationLayer.hide();
   }
 
-  private void updateMapWithOptions(LocationLayerOptions options) {
```

Comment:
We cannot simply remove this method. We need to call it in `LocationLayerPlugin#applyStyle` or move its logic there. Otherwise, those values won't be set on the map.

Reply:
Good catch, I readded the `updateMapWithOptions` method but without the checks since inside LocationLayerOptions we check the values.